### ISSUE TITLE
README: Please provide some details about the .dropboxignore file

### ISSUE THREAD

**Issue by birgersp** [sp1thas/dropboxignore#25]
Hello,

Sorry but I am confused. 

The .dropboxignore file specifies which files to be ignored by dropbox sync. But is this a (built-in) feature of dropbox? Or is it a feature added by this application (project).

If it's a feature added by this project, I am confused why that isn't explicitly stated in the README (because .dropboxignore is what I was looking for). The CLI tool is nice and all, but all I personally (and probably others) care about is adding stuff to .dropboxignore. Namely,

```**/node_modules```

in `.dropboxignore` in the root directory of all my Node projects.

And I just did, and it works! Amazing!

I also tried `genupi`, but it created a plethera of .dropboxignore files. I'd rather not have to put one of those in all my projects.

TLDR: Explain the `.dropboxignore` file in README.

**Comment by sp1thas** [comment]
Hey @birgersp , firstly, thanks for your feedback. The ability to ignore a file or folder from dropbox is a [built-in feature](https://help.dropbox.com/files-folders/restore-delete/ignored-files). On the other hand, the `.dropboxignore` file is not natively supported by dropbox (that's why I started this project). I think you have a point, the concept of `.dropboxignore` file could be a bit more detailed. Are you willing to open a PR for this?

**Comment by birgersp** [comment]
In all honesty, I moved to MEGA instead of Dropbox. Initially just because MEGA has a good feature for excluding files/folders (using pattern matching, wildcards etc).
